addons/web/static/src/views/form/button_box/button_box.js

```diff
@@ -17,7 +17,10 @@ export class ButtonBox extends Component {
     setup() {
         const ui = useService("ui");
         onWillRender(() => {
-            const maxVisibleButtons = [3, 4, 5, 7, 4, 5, 8][ui.size] || 8;
+            let maxVisibleButtons = [3, 4, 5, 7, 4, 5, 8][ui.size] || 8;
+            if (ui.isSmall) {
+                maxVisibleButtons = 0;
+            }
```

Comment:
```suggestion
            let maxVisibleButtons = [0, 0, 0, 7, 4, 5, 8][ui.size] || 8;
```
Will this works ?
As UI Service size =
```javascript
SIZES = { XS: 0, VSM: 1, SM: 2, MD: 3, LG: 4, XL: 5, XXL: 6 };
```

Reply:
Doesn't work because `0` is falsy... and then it falls back to `8` :sweat_smile:

Reply:
And
```suggestion
            let maxVisibleButtons = [0, 0, 0, 7, 4, 5, 8][ui.size] ?? 8;
```